### ISSUE TITLE
How to run a pretrained hierarichal story gen model on a test test to get loss and ppl?

### ISSUE THREAD

**Issue by Hannabrahman** [pytorch/fairseq#1025]
Hi,

I have trained a story generation model and got the train/validatation score. I wonder  if there is am easy way of running the this trained model on test and get the test loss and perplexity?

Thanks

**Comment by weiweitoo** [comment]
@Hannabrahman I am facing this problem as well. Let me know if you solved it

**Comment by huihuifan** [comment]
You should be able to run generate.py with --score-reference and --gen-subset test
